app/assets/stylesheets/overrides.scss

```diff
@@ -10,12 +11,34 @@ $navactive: $orange;
 $tabdown: $cream;
 $font: $brown;
 
+.container {
+background-color: #fff;
+border-radius: 20px ;
```

Comment:
Line should be indented 2 spaces, but was indented 0 spaces<br>Declaration should not have a space before the terminating semicolon